setup.py

```diff
@@ -143,11 +143,11 @@
             "pytest-xdist~=1.29",
             "tox~=3.14",
             "autopep8~=1.4",
+            "black~=20.8b1; python_version >= '3.6'",
```

Comment:
I assumed that an update to the latest version would not be a problem!

Reply:
black formats differently based on its version, so black should probably be pinned in the projects using this library, not here. If it must be here, then probably having it be any version makes more sense?

Reply:
@Madoshakalaka thoughts? This issue (#34) is causing problems for us.

Reply:
@grizz I've moved the requirement to the dev requirements in this PR, so it won't attempt to install black at all when you use it.

Reply:
I'm also running into this issue. Wouldn't upgrading the `dev` black dependency require reformatting this code base (though I suppose a follow-up PR would be appropriate for that)? 

Aside from that, I def like this solution, FWIW.